src/Vm.sol

```diff
@@ -188,156 +249,135 @@ interface VmSafe {
+    // Sets block.difficulty
+    function difficulty(uint256 newDifficulty) external;
     // Sets block.chainid
-    function chainId(uint256) external;
-    // Stores a value to an address' storage slot, (who, slot, value)
-    function store(address, bytes32, bytes32) external;
+    function chainId(uint256 newChainId) external;
+    // Stores a value to an address' storage slot.
+    function store(address who, bytes32 slot, bytes32 value) external;
     // Sets the nonce of an account; must be higher than the current nonce of the account
-    function setNonce(address, uint64) external;
+    function setNonce(address account, uint64 newNonce) external;
     // Sets the *next* call's msg.sender to be the input address
-    function prank(address) external;
+    function prank(address msgSender) external;
     // Sets all subsequent calls' msg.sender to be the input address until `stopPrank` is called
-    function startPrank(address) external;
+    function startPrank(address msgSender) external;
     // Sets the *next* call's msg.sender to be the input address, and the tx.origin to be the second input
-    function prank(address, address) external;
+    function prank(address msgSender, address txOrigin) external;
     // Sets all subsequent calls' msg.sender to be the input address until `stopPrank` is called, and the tx.origin to be the second input
-    function startPrank(address, address) external;
+    function startPrank(address msgSender, address txOrigin) external;
     // Resets subsequent calls' msg.sender to be `address(this)`
     function stopPrank() external;
-    // Sets an address' balance, (who, newBalance)
-    function deal(address, uint256) external;
-    // Sets an address' code, (who, newCode)
-    function etch(address, bytes calldata) external;
+    // Sets an address' balance
+    function deal(address who, uint256 newBalance) external;
+    // Sets an address' code
+    function etch(address who, bytes calldata newCode) external;
     // Expects an error on next call
-    function expectRevert(bytes calldata) external;
-    function expectRevert(bytes4) external;
+    function expectRevert(bytes calldata revertData) external;
+    function expectRevert(bytes4 revertData) external;
     function expectRevert() external;
     // Prepare an expected log with (bool checkTopic1, bool checkTopic2, bool checkTopic3, bool checkData).
     // Call this function, then emit an event, then call a function. Internally after the call, we check if
     // logs were emitted in the expected order with the expected topics and data (as specified by the booleans)
-    function expectEmit(bool, bool, bool, bool) external;
-    function expectEmit(bool, bool, bool, bool, address) external;
+    function expectEmit(bool checkTopic1, bool checkTopic2, bool checkTopic3, bool checkData) external;
+    function expectEmit(bool checkTopic1, bool checkTopic2, bool checkTopic3, bool checkData, address emitter)
```

Comment:
```suggestion
    function expectEmit(bool checkTopic1, bool checkTopic2, bool checkTopic3, bool checkData, address target)
```

Reply:
I went with `emitter`, this is what's used in the Foundry Book. And I just like it better (nit).